rmf_fleet_msgs/msg/Location.msg

```diff
@@ -2,5 +2,9 @@ builtin_interfaces/Time t
 float32 x
 float32 y
 float32 yaw
+
+# Target speed of the lane leading to this waypoint, to be respected if > 0.0
+float32 approach_speed
```

Comment:
Since the field is about speed limits, I would hesitate to rely on zero initialization. Could we have the default value to be -1 instead, and change the field name to be `approach_speed_limit` too?

The documentation makes it sound like the robots **must** be travelling at this speed approaching this location.